include_core/omrport.h

```diff
@@ -1242,7 +1242,11 @@ typedef struct J9PlatformStackFrame {
 } J9PlatformStackFrame;
 
 typedef struct J9PlatformThread {
+#if defined(OSX)
+	uint64_t thread_id;
+#else /* defined(OSX) */
 	uintptr_t thread_id;
+#endif /* defined(OSX) */
```

Comment:
OSX is a 64-bit only platform so this doesn't really change anything. I think this can safely be reverted.